Library/Formula/zeromq.rb

```diff
@@ -28,6 +28,12 @@ class Zeromq < Formula
   depends_on "libpgm" => :optional
   depends_on "libsodium" => :optional
 
+  if not build.head?
+    # enable --without-libsodium on libzmq < 4.1
+    # zeromq/zeromq4-x#105
+    patch :DATA
```

Comment:
Thinking about it, you might get asked to create a ` stable do ` block with the stable url, checksum and this patch in. ` stable do ` tends to be preferred over ` if build.stable? patch do/:DATA ` these days.